src/axom/quest/SignedDistance.hpp

```diff
@@ -109,25 +109,45 @@ class SignedDistance
    * \brief Creates a SignedDistance instance for queries on the given mesh.
    * \param [in] surfaceMesh user-supplied surface mesh.
    * \param [in] isWatertight indicates if the surface mesh is closed.
-   * \param [in] maxObjects max number of objects for spatial decomposition.
-   * \param [in] maxLevels max levels for spatial decomposition.
    * \param [in] computeSign indicates if distance queries should compute signs (optional).
+   * \param [in] allocatorID the allocator to create the underlying BVH with.
```

Comment:
the allocator id arg is also defaulted so is optional. please note that for consistency.